lib/msf/core/reflective_dll_loader.rb

```diff
@@ -48,17 +52,26 @@ def load_rdi_dll_from_data(dll_data)
 
   private
 
-  def parse_pe(dll)
+  def parse_pe(dll, loader_name: 'ReflectiveLoader', loader_ordinal: EXPORT_REFLECTIVELOADER)
     pe = Rex::PeParsey::Pe.new(Rex::ImageSource::Memory.new(dll))
     offset = nil
 
-    pe.exports.entries.each do |e|
-      if e.name =~ /^\S*ReflectiveLoader\S*/
-        offset = pe.rva_to_file_offset(e.rva)
-        break
+    unless loader_name.nil?
+      pe.exports.entries.each do |e|
+        if e.name =~ /^\S*${loader_name}\S*/
+          offset = pe.rva_to_file_offset(e.rva)
+          break
+        end
       end
     end
 
+    # If we aren't able to find the ReflectiveLoader, we need to
+    # fallback to the knownn ordinal export for RDI DLLs?
```

Comment:
Minor typo here in the comments.

```suggestion
    # fallback to the known ordinal export for RDI DLLs?
```

Reply:
Thanks!